src/planner/update_plan.cpp

```diff
@@ -118,7 +119,7 @@ void UpdatePlan::BuildInitialUpdatePlan(
 UpdatePlan::UpdatePlan(const parser::UpdateStatement *parse_tree)
     : update_primary_key_(false) {
   std::vector<oid_t> column_ids;
-  BuildInitialUpdatePlan(parse_tree, column_ids);
+  BuildInitialUpdatePlan(parse_tree, column_ids, nullptr);
```

Comment:
Above BuildInitialUpdatePlan, annotation says "FIXME: Should remove when the simple_optimizer tears down". And I think this function is outdated. Peloton won't use it anymore. So it doesn't matter.